grails-app/assets/javascripts/streama/controllers/admin-reports-ctrl.js

```diff
@@ -0,0 +1,48 @@
+//= wrapped
+
+angular.module('streama').controller('adminReportsCtrl', [
+  'apiService', '$state', '$rootScope', function (apiService, $state, $rootScope) {
+    var vm = this;
+    vm.selectedReports = [];
+    vm.resolveReports = resolveReports;
+    vm.addOrRemoveFromSelection = addOrRemoveFromSelection;
+    apiService.video.getErrorReports().then(function (reports) {
+      vm.reports = reports.data;
+      vm.test = 'test';
+      console.log(vm.reports);
+    });
+
+    function addOrRemoveFromSelection($event, report) {
+      if($event.target.checked) {
+        vm.selectedReports.push(report.id);
+      } else {
+        _.remove(vm.selectedReports, function(id) {
+          return id === report.id;
+        });
+      }
+    }
+    function resolveReports() {
+      if(vm.selectedFiles.length > 0) {
```

Comment:
Haha this was during one of the earlier commits I believe right after I copied some lines from the file manager and forgot to change files to reports. Learned here that it can be easier to just write the code again yourself instead of copying.